docs/manuals/source/Configuration/CustomizingTheConfiguration.rst

```diff
@@ -450,10 +451,10 @@ Data Types
 
 When parsing the resource directives, Bareos classifies the data according to the types listed below.
 
-.. _DataTypeAcl:
+.. _DataTypeACL:
 
-acl
-   :index:`\ <single: Data Type; acl>`\
+ACL
+   :index:`\ <single: Data Type; ACL>`\
```

Comment:
```suggestion
   .. index::
      single: Data Type; ACL
```
An index is better written in this format. The other format has been introduced by the automatic conversion from Latex and is only required for an inline index.
If you adapt this section, best use this format, see https://docs.bareos.org/DocumentationStyleGuide/RestOverview.html#index